app/src/renderer/components/common/TmBalance.vue

```diff
@@ -51,7 +51,7 @@ export default {
     }
   },
   computed: {
-    ...mapGetters([`bondingDenom`, `user`, `totalAtoms`]),
```

Comment:
what is the thinking here @fedekunze ? isn't it nicer to have `bondingDenom` as a getter than having to use `stakingParameters.parameters.bond_denom` every time?

Reply:
hmmm just deleted it bc is was calling config.denom which was a mocked value. Tbh I didn't thought much about it. Do you think we need a `getter` for it ?

Reply:
updated